app/component/app-edit-gallery-item-form/app-edit-gallery-item-form-directive.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const angular = require('angular');
+angular.module('galleryApp')
+.directive('appEditGalleryItemForm', function() {
+  return {
+    restrict: 'E',
+    replace: true,
+    scope: {
+      title:  '=',
```

Comment:
You'll have an easier time if you pass the whole image through rather than each of its properties separately.